CONTRIBUTING.md

```diff
@@ -14,8 +14,6 @@ This page provides information about contributing code to the Jenkins core codeb
    - Apache Maven 3.8.1 or above. You can [download Maven here](https://maven.apache.org/download.cgi).
      In the Jenkins project we usually use the most recent Maven release.
    - Any IDE which supports importing Maven projects.
-   - Install [NodeJS 16.x](https://nodejs.org/en/). **Note:** only needed to work on the frontend assets found in the `war` module.
-     - Frontend tasks are run using [yarn](https://yarnpkg.com/). Run `npm install -g yarn` to install it.
```

Comment:
it is needed for the `Building frontend assets` section though I think. the maven frontend node / yarn won't be in the users path.

Reply:
I see, thought people would build Jenkins first, before doing anything else particularly.

Quickly changed the purpose of this PR then :P